frontend/src/pages/annotate.js

```diff
@@ -390,6 +391,9 @@ class Annotate extends React.Component {
             </div>
             {!isDataLoading ? (
               <div>
+                <div className="row justify-content-md-center my-4">
```

Comment:
Two things here.

1. We should show filename at the top of the annotation dashboard.
2. We should show `original_filename` and not hashed one.